controllers/api.controllers.js

```diff
@@ -36,12 +36,19 @@ exports.getReviewWithComments = (req, res, next) => {
     const { review_id } = req.params
 
     return selectReviewWithComments(review_id).then((comments) => {
-        console.log(comments)
         if (comments) {
             res.status(200).send({ comments: comments })
-
-
         }
     }).catch(next)
 
 }
+
+exports.postComment = (req, res, next) => {
+    const newComment = req.body;
+    const { review_id } = req.params
+
+    return addComment(newComment, review_id).then((newComment) => {
+        return res.status(201).send({ newComment })
+    }).catch(next)
```

Comment:
I've just spotted this but you should have a catch block with an arrow function and a parameter of err. Then you invoke next with err like so next(err). Please do this for other controllers as well